assets/helpers/internationalisation/country.js

```diff
@@ -434,15 +431,25 @@ function handleCountryForCountryGroup(
 }
 
 function detect(countryGroupId: ?CountryGroupId = null): IsoCountry {
-  const country = handleCountryForCountryGroup('EURCountries', countryGroupId) ||
-                  handleCountryForCountryGroup('International', countryGroupId) ||
-                  handleCountryForCountryGroup('NZDCountries', countryGroupId) ||
-                  fromCountryGroup(countryGroupId) ||
-                  fromPath() ||
-                  fromQueryParameter() ||
-                  fromCookie() ||
-                  fromGeolocation() ||
-                  'GB';
+
+  const targetCountryGroups = ['International', 'EURCountries', 'NZDCountries', 'GBPCountries', 'AUDCountries'];
```

Comment:
This array could also use the declared type ^.

Reply:
I created a type to group these countries. When we are able to include AU on the list there is room for further refactor and replace this type with countryGroupId.